cli/internal/cmd/version.go

```diff
@@ -58,18 +60,22 @@ func parseBuildInfo(info *debug.BuildInfo) (commit, state, date, goVersion, comp
 			os = value
 		case "vcs.time":
 			date = value
-		case "vcs.modified":
-			if value == "true" {
-				state = "dirty"
-			} else {
-				state = "clean"
-			}
```

Comment:
Why remove the modified flag? If we have custom stamping tools for Bazel, can we not reimplement this part easily for the Bazel stamping?

Reply:
I felt like it only caused issues in our CI when non-ignored files landed in the workdir during a build which marked the release CLI as dirty. I can add it back if you feel like it provides value.

Reply:
>when non-ignored files landed in the workdir during a build 

Which shouldn't happen, given that this already resulted in funny bugs (e.g. wrong SBOMs). 
Thus yes, I'd say it provides value :)